src/user/serializers.py

```diff
@@ -1050,6 +1050,7 @@ def get_coauthor(self, coauthor):
 class DynamicAuthorProfileSerializer(DynamicModelFieldSerializer):
     institutions = SerializerMethodField()
     coauthors = SerializerMethodField()
+    reputations = SerializerMethodField()
```

Comment:
Could we change name to `reputation_list` or a different name? I'm thinking we will also need a field called `reputation` to indicate what the user's rep is (atm it is decided it will be the highest value in that list: `max_val(reputations)`